source/opt/compact_ids_pass.cpp

```diff
@@ -21,6 +21,22 @@
 
 namespace spvtools {
 namespace opt {
+namespace {
+
+uint32_t GetRemappedId(
```

Comment:
Please add comments describing this function.

Reply:
Ah right. Done.